crowbar_framework/app/models/service_object.rb

```diff
@@ -1297,9 +1297,11 @@ def apply_role(role, inst, in_queue, bootstrap = false)
 
       next if bad_nodes.empty?
 
-      message = "Failed to apply the proposal to: "
+      message = "Failed to apply the proposal to "
```

Comment:
also, I wouldn't change that line as things can be a bit complicated in case many nodes have a failure, and the `:` make it clearer that it may be a list.